src/components/List_Products.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { saveItem } from '../lib/api';
+import { db } from '../lib/firebase';
+import { collection, onSnapshot } from 'firebase/firestore';
```

Comment:
agreeing on a way to sort imports might be a good idea. Just for reference: https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m you don't have to add eslint but just a guide

Reply:
Wow! I didn't know about it. Thanks for sharing this post! 😄